### PR TITLE
Sick LD-MRS

### PR DESCRIPTION
**Pull request by fabienrohrer**
Result: https://www.cyberbotics.com/doc/automobile/lidar-sensors?version=enhancement-sick-ld-mrs#sick-ld-mrs
Dual PR: https://github.com/omichel/webots/pull/6651

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

File: automobile/lidar-sensors.md [hunk 1]
```diff
@@ -64,12 +67,51 @@ SickLms291 {
 }
 ```
 
-The `noise` field specifies the standard deviation of the gaussian noise in
+The `noise` field specifies the standard deviation of the gaussian depth noise in
```
Comment (omichel) [inline automobile/lidar-sensors.md]: Shouldn't we say "depth gaussian noise" instead?

File: automobile/lidar-sensors.md [hunk 2]
```diff
@@ -64,12 +67,51 @@ SickLms291 {
...
+  SFRotation rotation 0 1 0 0
+  SFString name "LD-MRS"
+  SFFloat noise 0.3
+  SFInt32 numberOfLayers 4
+  SFFloat angularResolution 0.008726646259972
+  SFBool physics TRUE
+}
+```
+
+The `noise` field specifies the standard deviation of gaussian image noise in
```
Comment (omichel) [inline automobile/lidar-sensors.md]: &rarr; "image gaussian noise"?

File: automobile/lidar-sensors.md [hunk 3]
```diff
@@ -64,12 +67,51 @@ SickLms291 {
...
+meters.
```
Comment (omichel) [inline automobile/lidar-sensors.md]: I wouldn't write "meters." on a new line.

File: automobile/lidar-sensors.md [hunk 4]
```diff
@@ -64,12 +67,51 @@ SickLms291 {
 }
 ```
 
-The `noise` field specifies the standard deviation of the gaussian noise in
+The `noise` field specifies the standard deviation of the gaussian depth noise in
 meters.
```
Comment (omichel) [inline automobile/lidar-sensors.md]: I wouldn't write "meters." on a new line.

File: automobile/lidar-sensors.md [hunk 5]
```diff
@@ -64,12 +67,51 @@ SickLms291 {
...
 
 The `resolution` field specifies the number of points returned per layer per
 scan.
 
+#### Sick LD-MRS
+
+The `Sick LD-MRS` is a 2 or 4 layers lidar with a range of 300 meters and a
+field of view of respectively 110 or 85 degrees.
```
Comment (omichel) [inline automobile/lidar-sensors.md]: We should write one sentence per line and one line per sentence in the MD files.

File: automobile/lidar-sensors.md [hunk 6]
```diff
@@ -64,12 +67,51 @@ SickLms291 {
...
+
+The top and bottom layers are split horizontally with an angle of 2.4 degrees.
+Layer 0 corresponds to the bottom layer.
+First response values are corresponding to the device right.
+The frustum cone is shifted to the right by an offset angle of 7.5 degrees when 4 layers are set,
+and 5 degrees when 2 layers are set.
```
Comment (omichel) [inline automobile/lidar-sensors.md]: one line per sentence.

File: automobile/lidar-sensors.md (hunk 1 above)
Comment (tn12787) [reply automobile/lidar-sensors.md]: It shouldn't, the noise applies to the depth values, and its distribution is gaussian, so gaussian serves as an adjective here and should come first.

Comment (fabienrohrer) [reply automobile/lidar-sensors.md]: Thank you. Then I think this file is entirely correct related with this.

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**